drivers/wifi/infineon/Kconfig.airoc

```diff
@@ -4,19 +4,37 @@
 
 menuconfig WIFI_AIROC
 	bool "Infineon AIROC SoC Wi-Fi support"
+	depends on DT_HAS_INFINEON_AIROC_WIFI_ENABLED
+	default y
 	select THREAD_CUSTOM_DATA
 	select WIFI_OFFLOAD
+	select NET_L2_ETHERNET
 	select NET_L2_WIFI_MGMT
-	select SDIO_STACK
-	select SDHC
 	select WIFI_USE_NATIVE_NETWORKING
 	select USE_INFINEON_ABSTRACTION_RTOS
-	depends on DT_HAS_INFINEON_AIROC_WIFI_ENABLED
+	select SDHC if $(dt_compat_on_bus,$(DT_COMPAT_INFINEON_AIROC_WIFI),sd)
+	select SDIO_STACK if $(dt_compat_on_bus,$(DT_COMPAT_INFINEON_AIROC_WIFI),sd)
+	select SPI if $(dt_compat_on_bus,$(DT_COMPAT_INFINEON_AIROC_WIFI),spi)
```

Comment:
move `select SDHC`, `select SDIO_STACK`, `select SPI` to  `config  AIROC_WIFI_BUS_SPI` and `config AIROC_WIFI_BUS_SDIO`

Reply:
Done.